### PR TITLE
Remove Appcelerator references

### PR DESCRIPTION
**Pull request by sedroche**
As per ticket [FH-3250](https://issues.jboss.org/browse/FH-3250) we are no longer supporting Titanium/appcelerator apps. Removing all references in the repos also

@pmdarrow

### CODE REVIEW

**Comment by jcesarmobile** [comment]
I found this about AeroGear-Push-Titanium https://github.com/feedhenry/feedhenry.github.io/blob/master/_data/feedhenry-repos.json#L2-L8

It's not displayed on the website, but should be good to remove it

**Review by pb82 — APPROVED** [review]
LGTM. ping @pmdarrow

File: _data/feedhenry-repos.json [hunk 1]
```diff
@@ -1,11 +1,4 @@
 {
-    "AeroGear-Push-Titanium": {
```
Comment (pmdarrow) [inline _data/feedhenry-repos.json]: This data will just get pulled back in when `_get_repo_data.py` is run again, so I'd remove this change. As long as you're not referencing "AeroGear-Push-Titanium" in any templates, it won't get shown.

**Review by pmdarrow — CHANGES_REQUESTED** [review]
Just one small fix, otherwise LGTM!

**Review by pmdarrow — APPROVED** [review]

**Pull request merged**